Qfront/src/redux/api.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+// Create base url for axios to
+const axiosInstance = axios.create({
```

Comment:
You can export this `const` as a named export at the end of the file, instead of reassigning it to another variable and export that